sigs.yml

```
@@ -0,0 +1,502 @@
+    invites: calendar-ruby
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Rust: SDK'
+    meeting: Every Tuesday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4
+    chat:
+      - type: slack
+        name: '#otel-rust'
+        id: C03GDP0H023
+      - type: other
+        name: Gitter
+        link: https://gitter.im/open-telemetry/opentelemetry-rust
+    invites: calendar-rust
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Swift: SDK'
+    meeting: Every Thursday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1LugL8r4bAkbTxZ1Gq_6j_8SDv1LuZaNeUdR-QLN4d48
+    chat:
+      - type: slack
+        name: '#otel-swift'
+        id: C01NCHR19SB
+    invites: calendar-swift
+    tcSponsors: []
+    gcLiaison: []
+  - name: Communications (Website, [Documentation](https://opentelemetry.io/docs/),
+      etc.)
+    meeting: Every other week on Monday at 10:00 PT
+    notes:
+      type: gDoc
+      value: 1wW0jLldwXN8Nptq2xmgETGbGn9eWP8fitvD5njM-xZY
+    chat:
+      - type: slack
+        name: '#otel-comms'
+        id: C02UN96HZH6
+    invites: calendar-comms
+    tcSponsors: []
+    gcLiaison:
+    - name: 'Severin Neumann'
+      github: svrnm
+  - name: End-User WG
+    meeting: Every other week on Thursday at 10:00 PT
+    notes:
+      type: gDoc
+      value: 1e-UNZA3Tuno9b53RQbe--whUcO0VIXF3P81oXsrBK6g
+    chat:
+      - type: slack
+        name: '#otel-user-research'
+        id: C01RT3MSWGZ
+    invites: calendar-user-research
+    tcSponsors: []
+    gcLiaison:
+    - name: Daniel Gomez Blanco
+      github: danielgblanco
+  - name: eBPF
+    meeting: Every week on Tuesday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 13GK915hdDQ9sUYzUIWi4pOfJK68EE935ugutUgL3yOw
+    chat:
+      - type: slack
+        name: '#otel-ebpf'
+        id: C02AB15583A
+    invites: calendar-ebpf
+    tcSponsors: []
+    gcLiaison: []
+  - name: Agent Management WG
+    meeting: Every other week on Tuesday at 11:00 PT
+    notes:
+      type: gDoc
+      value: 19WA5-ex8rNFIBIyVb5VqMXfWNmUQwppGhN8zBeNG0f4
+    chat:
+      - type: slack
+        name: '#otel-agentmanwg'
+        id: C02J58HR58R
+    invites: calendar-agent-mgmt
+    tcSponsors: []
+    gcLiaison: []
+  - name: Prometheus Interoperability WG
+    meeting: Every other week on Wednesday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8
+    chat:
+      - type: slack
+        name: '#otel-prometheus-wg'
+        id: C01LSCJBXDZ
+    invites: calendar-prometheus
+    tcSponsors: []
+    gcLiaison: []
+  - name: Client Instrumentation
+    meeting: Every Tuesday at 9:00 AM PT
+    notes:
+      type: gDoc
+      value: 16Vsdh-DM72AfMg_FIt9yT9ExEWF4A_vRbQ3jRNBe09w
+    chat:
+      - type: slack
+        name: '#otel-client-side-telemetry'
+        id: C0239SYARD2
+    invites: calendar-client-side
+    tcSponsors: []
+    gcLiaison: []
```

```suggestion
    gcLiaison: []
```
@open-telemetry/governance-committee please assign yourself for being the liaison person for this SIG

---

Proposed changed below